onedal/linear_model/incremental_linear_model.py

```diff
@@ -230,24 +235,33 @@ def partial_fit(self, X, y, queue=None):
         self : object
             Returns the instance itself.
         """
+        if not hasattr(self, "_params"):
+            self._params = self._get_onedal_params(X.dtype)
+
         module = self._get_backend("linear_model", "regression")
 
+        self._sua_iface, self._xp, _ = _get_sycl_namespace(X)
+        use_raw_input = _get_config().get("use_raw_input", False) is True
+        if use_raw_input and self._sua_iface is not None:
+            queue = X.sycl_queue
+
         self._queue = queue
         policy = self._get_policy(queue, X)
 
-        X, y = _check_X_y(
-            X, y, dtype=[np.float64, np.float32], accept_2d_y=True, force_all_finite=False
-        )
-        y = np.asarray(y, dtype=X.dtype)
+        if not use_raw_input:
```

Comment:
There is no spmd ridge support, so I am going to remove raw inputs support here. just FYI